frontend/dockerui/namedcontext.go

```diff
@@ -58,15 +59,21 @@ func (bc *Client) namedContext(ctx context.Context, name string, nameWithPlatfor
 
 		named = reference.TagNameOnly(named)
 
-		dgst, data, err := bc.client.ResolveImageConfig(ctx, named.String(), llb.ResolveImageConfigOpt{
+		ref, dgst, data, err := bc.client.ResolveImageConfig(ctx, named.String(), llb.ResolveImageConfigOpt{
```

Comment:
This code is also used by external frontends that can run against old buildkit so returning `ref` should be considered optional. Another way would be to handle this in the `grpcclient` level and fill in input name for old implementations.